src/algorithms/peps_opt.jl

```diff
@@ -157,6 +157,13 @@ function fixedpoint(
         finalize! = (x, f, g, numiter) -> fin!(symm_finalize!(x, f, g, numiter)..., numiter)
     end
 
+    if scalartype(C) <: Real || scalartype(T) <: Real
+        env₀ = CTMRGEnv(complex.(env₀.corners), complex.(env₀.edges))
```

Comment:
```suggestion
        env₀ = complex(env₀)
```
Probably cleaner to just define `complex` for the environments

Reply:
Yes, agree. I also defined `real` for future use